src/main/java/org/elasticsearch/cluster/metadata/IndexMetaData.java

```diff
@@ -784,4 +786,23 @@ public static void writeTo(IndexMetaData indexMetaData, StreamOutput out) throws
             }
         }
     }
+
+    /**
+     * Returns <code>true</code> iff the given settings indicate that the index associated
+     * with these settings allocates it's shards on a shared filesystem. Otherwise <code>false</code>. The default
+     * setting for this is the returned value from {@link #isIndexUsingShadowReplicas(org.elasticsearch.common.settings.Settings)}.
+     */
+    public static boolean usesSharedFilesystem(Settings settings) {
```

Comment:
can we use the same method structure between this method and the following? I like `isIndex...`